docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -57,6 +57,39 @@ Property name                                             Required   Description
+                                                                     number of requests made to Pinot. This is useful for smaller Pinot clusters,
+                                                                     default is ``1``.
+``pinot.fetch-retry-count``                               No         Retry count for retriable Pinot data fetch calls, default is ``2``.
+``pinot.non-aggregate-limit-for-broker-queries``          No         Max limit for non aggregate queries to the Pinot broker, default is ``25,000``.
+``pinot.max-rows-for-broker-queries``                     No         Max rows for a broker query can return, default is ``50,000``.
+``pinot.aggregation-pushdown.enabled``                    No         Push down aggregation queries, default is ``true``.
+``pinot.count-distinct-pushdown.enabled``                 No         Push down count distinct queries to Pinot, default is ``true``.
+``pinot.target-segment-page-size``                        No         Max allowed page size for segment query, default is ``1MB``.
+========================================================= ========== ==============================================================================
+
+gRPC configuration properties:
+
+========================================================= ========== ==============================================================================
+Property name                                             Required   Description
+========================================================= ========== ==============================================================================
+``pinot.grpc.enabled``                                    No         Use gRPC endpoint for Pinot server queries, default is ``true``.
+``pinot.grpc.port``                                       No         Pinot gRPC port, default to ``8090``.
+``pinot.grpc.max-inbound-message-size``                   No         Max inbound message bytes when init gRPC client, default is ``128MB``.
+``pinot.grpc.use-plain-text``                             No         Use plain text for gRPC communication, default to ``true``.
+``pinot.grpc.tls.keystore-type``                          No         TLS keystore type for gRPC connection, default is ``JKS``.
+``pinot.grpc.tls.keystore-path``                          No         TLS keystore file location for gRPC connection, default is empty.
+``pinot.grpc.tls.keystore-password``                      No         TLS keystore password, default is empty.
+``pinot.grpc.tls.truststore-type``                        No         TLS truststore type for gRPC connection, default is ``JKS``.
+``pinot.grpc.tls.truststore-path``                        No         TLS truststore file location for gRPC connection, default is empty.
+``pinot.grpc.tls.truststore-password``                    No         TLS truststore password, default is empty.
+``pinot.grpc.tls.ssl-provider``                           No         TLS ssl provider, default is ``JDK``.
```

Comment:
```suggestion
``pinot.grpc.tls.ssl-provider``                           No         SSL provider, default is ``JDK``.
```

Reply:
It would be nice if you can list possible values or add a link to official documentation. Follow-up PR is fine though.